mbuf/mbuf.go

```diff
@@ -43,6 +49,46 @@ func PktMbufClone(m *Mbuf, p *mempool.Mempool) {
 	C.rte_pktmbuf_clone(mbuf(m), mp(p))
 }
 
+// PktMbufAlloc allocate an uninitialized mbuf from mempool p.
+func PktMbufAlloc(p *mempool.Mempool) *Mbuf {
```

Comment:
`NULL` may be returned here. How do you suggest to handle it? Document `NULL` or return `(mbuf, err)` tuple?